nixos/modules/services/web-servers/caddy/default.nix

```
@@ -35,7 +35,8 @@ let
 
       Caddyfile-formatted = pkgs.runCommand "Caddyfile-formatted" { nativeBuildInputs = [ cfg.package ]; } ''
         mkdir -p $out
-        ${cfg.package}/bin/caddy fmt ${Caddyfile}/Caddyfile > $out/Caddyfile
+        cp --no-preserve=mode ${Caddyfile}/Caddyfile $out/Caddyfile
+        ${cfg.package}/bin/caddy fmt --overwrite $out/Caddyfile
```

```suggestion
        caddy fmt --overwrite $out/Caddyfile
```
As caddy is already in nativeBuildInputs.